### PR TITLE
Preliminary water system component class; pending subdivision

### PR DESCRIPTION
**Pull request by LifesaCode**
Have a look, I found another infographic that added some things my initial diagram was missing, as well as a new additive that increased production significantly. This will need to be subdivided; probably  urine treatment portion in one class and everything else in another. 

Add comments and/or questions, and I will respond to them later this evening.

### CODE REVIEW

File: src/Orbit/Models/ThermalSystem.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    {
+        double actualCabinTemp;
+        double selectedCabinTemp;
+
+        double internalCoolantTemp;
+        bool internalCoolantPumpOn;
+        double internalCoolantPumpTemp;
+        double internalCoolantPumpVibration;
+        int internalCoolantTankLevel;
+        bool cabinCoolantLeakSensorTripped;
```
Comment (josh-degraw) [inline src/Orbit/Models/ThermalSystem.cs]: Would it make sense to encapsulate these into another object?

File: src/Orbit/Models/ThermalSystem.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+        double externalCoolantTemp;
+        bool externalcoolantPumpOn;
+        double externalCoolantPumpTemp;
+        double externalCoolantPumpVibration;
+        int externalCoolantTankLevel;
```
Comment (josh-degraw) [inline src/Orbit/Models/ThermalSystem.cs]: Would it make sense to encapsulate these into another object too?

**Review by josh-degraw — CHANGES_REQUESTED** [review]
Before I merge this, could you just consolidate these types, i.e. by getting rid of the original ones you added that you've made updates for, and putting them in the `src/Orbit/Models/` directory under the namespace `Orbit.Models`? I'm not sure which pieces from your original code would need to stay or not. Thanks.

**Comment by LifesaCode** [comment]
Yes, give a little bit, just got home.

File: src/Orbit/Models/ThermalSystem.cs (hunk 1 above)
Comment (LifesaCode) [reply src/Orbit/Models/ThermalSystem.cs]: These were moved into InternalCoolantLoop.cs

File: src/Orbit/Models/ThermalSystem.cs (hunk 2 above)
Comment (LifesaCode) [reply src/Orbit/Models/ThermalSystem.cs]: These were moved into another class as well, ExternalCoolantLoop.cs

**Review by LifesaCode — COMMENTED** [review]
Changes made as requested.

**Review by LifesaCode — COMMENTED** [review]
Ok, so I was a dummy and didn't see that changes could be made within this conversation. So, I went and searched the files out and did it the hard way. Now it won't automatically fix here... but the changes are done.

**Review by josh-degraw — APPROVED** [review]

**Pull request merged**